### PR TITLE
Add preferred port feature and host option

### PR DESCRIPTION
**Pull request by JacopKane**
The main package has these features but CLI needed some update in order to catch up.

Also updated dependencies like `meow` to use flags etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Get an available port",
   "license": "MIT",
   "repository": "sindresorhus/get-port-cli",
   "author": {
     "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
-    "url": "sindresorhus.com"
+    "url": "https://sindresorhus.com/"
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes.

File: package.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "get-port-cli",
-  "version": "1.1.0",
+  "version": "2.0.0",
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes.

File: cli.js [hunk 3]
```diff
@@ -2,11 +2,19 @@
 'use strict';
 const meow = require('meow');
 const getPort = require('get-port');
+const meowTemplate = require('./meow-template');
```
Comment (sindresorhus) [inline cli.js]: It should be in the same file.

File: package.json [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
   "bin": {
     "get-port": "cli.js"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
   },
   "scripts": {
-    "test": "xo && ava"
+    "test": "xo --fix && ava"
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes.

File: package.json [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
     "email": "sindresorhus@gmail.com",
-    "url": "sindresorhus.com"
+    "url": "https://sindresorhus.com/"
   },
   "bin": {
     "get-port": "cli.js"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes.

File: package.json (hunk 1 above)
Comment (JacopKane) [reply package.json]: yes, thought about it but couldn't hold myself back, cause my editor was keep warning about it 😞

File: package.json (hunk 5 above)
Comment (JacopKane) [reply package.json]: This is not unrelated, the main package (`get-port`) requires this

File: package.json (hunk 4 above)
Comment (JacopKane) [reply package.json]: yes, reverting it

File: cli.js (hunk 3 above)
Comment (JacopKane) [reply cli.js]: I moved it to make an exclusive config for this file in `.editorconfig`, four space tabs on CLI is taking a lot of screen space

**Comment by JacopKane** [comment]
Updated the PR, waiting for your feedback on another two I answered

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by JacopKane** [comment]
> Thanks :)

Pleasure 🎩
